doc/extending/cop.txt

```diff
@@ -186,6 +199,19 @@ There are less methods to define for an Op than for a Type:
        is high or when theano compilation directory is shared by many
        process (like on a network file server on a cluster).
 
+    .. method:: get_context(node)
+
+       (optional) If defined, should return the runtime context the op
+       needs.  This context will be passed to the C code through the
+       variable named in `sub['context']`.  The variable is also
+       available for use in the code returned by
+       :meth:`c_init_code_struct`.  If it returns `None` this is
+       considered the same as if the method was not defined.
```

Comment:
No, that is a documentation bug.  None does not have special meaning.  I'll fix it.